.circleci/config.yml

```diff
@@ -94,8 +94,8 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/player
-
-      - run: bazel build --config=ci --config=skip-ios -- //...
+      # need to filter because //android/demo rules dont resolve
+      - run: bazel build --config=ci -- $(bazel query '//plugins/... except filter("ios|swiftui|jvm|android", //plugins/...)') $(bazel query '//core/... except filter("ios|swiftui|jvm|android", //core/...)') $(bazel query '//react/... except filter("ios|swiftui|jvm|android", //react/...)')
```

Comment:
@brocollie08 can you add this to your list? Or should we make a new issue for converting the build step to build common?

Reply:
i can do this